types/common.ts

```diff
@@ -30,10 +30,11 @@ export type CategoricalVariableType = {
 }
 
 export type ValueVariableType = {
+  discrete: boolean
   name: string
   description: string
-  minVal: number
-  maxVal: number
+  minVal: string
```

Comment:
I think we should try to preserve the type as number and rely on the "discrete" flag

Reply:
The text field returns a string to useForm (even with type="number") and in this simple version the discrete flag is set by checking for "." in the string. You can force useForm to return a number but 1.0 becomes 1 before the check, so I'm not sure how we would check for that without having the explicitly set "discrete" option from the UI. So... maybe change it back to number, when we make the better version with an option to set it from the UI?

Reply:
Agreed - we should change it back when discrete becomes an explicit option